.github/workflows/benchmark_large.yml

```diff
@@ -9,10 +9,20 @@
 name: Benchmark Large
 
 on:
+  pull_request: # DO NOT SUBMIT
   schedule:
     # Scheduled to run at 09:00 UTC and 21:00 UTC.
     - cron: '0 09,21 * * *'
   workflow_dispatch:
+    shard-count:
+      description: |
+        A device-name to integer mapping as a comma separated list.
+        Allows to assign a distinct shard count for each target device.
+        The reserved keyword `default` assigns a shard count to all target devices
+        that are not explicitly listed.
+      # Please keep the shard count default value in sync with jobs.build_e2e_test_artifacts.with.shard-count
+      default: a2-highgpu-1g=2,c2-standard-16=2,default=1
```

Comment:
I checked the log of `benchmark_large.yml`, it seems like `a2-highgpu-1g` is not sharded?

Reply:
Yes, the hard-coded override that you pointed out in https://github.com/openxla/iree/pull/14084#discussion_r1231340106

I will kick off another test run with this removed.

Reply:
This is working correctly now: https://github.com/openxla/iree/actions/runs/5288760050

(At least it spawned 4 jobs - 2 GPU / 2 CPU)